### PR TITLE
Improve `ext.asyncio.events`

### PR DESCRIPTION
**Pull request by bryanforbes**
### Description
Improve `ext.asyncio.events`

### Checklist

This pull request is:

- [ ] A documentation / typographical error fix
	- Good to go, no issue or tests are needed
- [X] A short code fix
	- please include the issue number, and create an issue if none exists, which
	  must include a complete example of the issue.  one line code fixes without an
	  issue and demonstration will not be accepted.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.   one line code fixes without tests will not be accepted.
- [ ] A new feature implementation
	- please include the issue number, and create an issue if none exists, which must
	  include a complete example of how the feature would look.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.

**Have a nice day!**

### CODE REVIEW

File: sqlalchemy-stubs/ext/asyncio/events.pyi [hunk 1]
```diff
@@ -1,5 +1,3 @@
-from .engine import AsyncConnectable as AsyncConnectable
-from .session import AsyncSession as AsyncSession
 from ...engine import events as engine_event
 from ...orm import events as orm_event
```
Comment (CaselIT) [inline sqlalchemy-stubs/ext/asyncio/events.pyi:2]: actually I'm not sure these are public

Comment (zzzeek) [reply sqlalchemy-stubs/ext/asyncio/events.pyi:2]: ¯\_(ツ)_/¯

Comment (CaselIT) [reply sqlalchemy-stubs/ext/asyncio/events.pyi:2]: I guess this file could be removed then

Comment (bryanforbes) [reply sqlalchemy-stubs/ext/asyncio/events.pyi:2]: The file should be kept so `mypy` doesn't try to check the actual python file. I'll remove the contents.

Comment (CaselIT) [reply sqlalchemy-stubs/ext/asyncio/events.pyi:2]: maybe the classes should be there?
we do have the normal ones:
https://github.com/sqlalchemy/sqlalchemy2-stubs/blob/12534116867b51781797425ef8b9aa14f24c6dfe/sqlalchemy-stubs/engine/events.pyi#L9

Comment (bryanforbes) [reply sqlalchemy-stubs/ext/asyncio/events.pyi:2]: I misunderstood what you were talking about. `X as Y` imports aren't exported, so `engine_event` and `orm_event` aren't exported as part of this module's properties.

Comment (CaselIT) [reply sqlalchemy-stubs/ext/asyncio/events.pyi:2]: nevermind I did not notice that the file did not end at line 4

**Pull request merged**